diff/parse.go

```diff
@@ -365,7 +365,6 @@ func (r *HunksReader) ReadHunk() (*Hunk, error) {
 		if r.hunk == nil {
 			// Check for presence of hunk header.
 			if !bytes.HasPrefix(line, hunkPrefix) {
-				panic("X")
```

Comment:
Good catch, that would've produced a pretty strange stack trace in that case :+1: